features/step_definitions/common_steps.rb

```diff
@@ -256,8 +256,9 @@
   end
 end
 
-Then /^I see '(.*)' text on the page/ do |expected_text|
-  all(:xpath, "//*[normalize-space() = '#{expected_text}']").length.should >= 1
+Then /^I (don't |)see '(.*)' text on the page/ do |negative, expected_text|
+  all(:xpath, "//*[normalize-space() = '#{expected_text}']").length.should >= 1 if negative.empty?
```

Comment:
I changed this as per your advice, previously I had the syntax copied from another step definition with an optional negative.